python/dlisio/ext/core.cpp

```diff
@@ -295,33 +295,46 @@ std::string fingerprint(const std::string& type,
     return ref.fingerprint();
 }
 
-struct fdatasize {
-    int src;
-    int dst;
-};
-
-fdatasize fdata_size(const char* fmt) {
-    const auto fmt_str = std::string(fmt);
-    const auto fmt_msg = "invalid format specifier in " + fmt_str;
-
-    int variable;
-    auto err = dlis_pack_varsize(fmt, &variable, nullptr);
-    if (err) {
-        throw std::invalid_argument(fmt_msg);
-    }
-
-    if (variable) {
-        throw dl::not_implemented(fmt_msg);
-    }
-
-    int src = 0;
-    int dst = 0;
-    err = dlis_pack_size(fmt, &src, &dst);
```

Comment:
Can (and should) dlis_pack_size and dlis_pack_varsize be removed now?

Reply:
I don't think so, they're still quite useful. While not useful when actually parsing records, they're useful for interrogating a few key properties from format strings. They can also be used when figuring out exactly what code path to take.